code/replication/object/ETUtils.py

```diff
@@ -0,0 +1,58 @@
+"""
+Utilities for using an ElemmentTree.
+"""
+
+import xml.etree.ElementTree as ET
+
+def textGetter(filename):
+	'''
+	A getter function for each article. When next is called, it will return the 
+	next article. The files are split by the </doc> tag, which is at the end 
+	of every article.
+	'''
+	with open(filename, errors='replace') as nmlFile:
+		text = ""
+		for line in nmlFile:
+			text += line
+			if "</doc>" in line:
```

Comment:
I assume it was checked that it never happens that tags are not alone on a separate line?